<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
    <meta http-equiv="x-ua-compatible" content="ie=edge" />
    <title>InfOmics</title>
    <!-- MDB icon -->
    <link rel="icon" href="img/mdb-favicon.ico" type="image/x-icon" />
    <!-- Font Awesome -->
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.11.2/css/all.css" />
    <!-- Google Fonts Roboto -->
    <link
      rel="stylesheet"
      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap"
    />
    <!-- MDB -->
    <link rel="stylesheet" href="css/mdb.min.css" />
    <link rel="stylesheet" href="css/custom.css" />
  </head>
  <body>
    <div class="head-section head-section-home">
      <nav class="navbar navbar-expand-lg">
        <div class="container-fluid ">
          <a class="navbar-brand" href="#">
            <img
              src="./img/homepage/Infoverona.png"
              height="100"
              alt="Lab Logo"
            />
          </a>
          <button
            class="navbar-toggler"
            type="button"
            data-mdb-toggle="collapse"
            data-mdb-target="#navbarSupportedContent"
            aria-controls="navbarSupportedContent"
            aria-expanded="false"
            aria-label="Toggle navigation"
          >
            <i class="fas fa-bars"></i>
          </button>

          <div class="collapse navbar-collapse" id="navbarSupportedContent">
            <ul class="navbar-nav me-auto mb-2 mb-lg-0">
              <li class="nav-item">
                <a class="nav-link active" aria-current="page" href="#">HOMEPAGE</a>
              </li>
              <li class="nav-item">
                <a class="nav-link" href="./pages/research.html">RESEARCH</a>
              </li>
              <li class="nav-item">
                <a class="nav-link" href="./pages/people.html">PEOPLE</a>
              </li>
              <li class="nav-item">
                <a class="nav-link" href="./pages/publications.html">PUBLICATIONS</a>
              </li>
              <li class="nav-item">
                <a class="nav-link" href="./pages/software.html">SOFTWARES</a>
              </li>
              <li class="nav-item">
                <a class="nav-link" href="./pages/contact.html">CONTACT US</a>
              </li>
              <li class="nav-item github">
                <a class="nav-link" href="https://github.com/InfOmics" target="_blank">
                  <span><i class="fab fa-github"></i></span>
                </a>
              </li>
            </ul>
          </div>
        </div>
      </nav>
      <!-- Navbar -->
    <div class="container-fluid presentation">
      <img src="./img/homepage/infomics_white.png" alt="InfOmics Logo">
      <div class="presentation-text">
        <span>InfOmics</span> <p>focus it's research mainly on biological 
          network modeling, perturbation and analysis and classification of 
          phenotypes by coding and non-coding expression.</p>
      </div>
    </div>
  </div>
  <div class="research">
    Our research goes from 
    Genome to Network analysis in order to understand biological systems 
    under specific conditions. In particular, we develop algorithms for 
    searching in labeled directed and undirected graphs or multigraphs. 
    Up to date, we have designed ones of the fastest algorithms for subgraph 
    database search, subgraph matching with or without redundancy on the 
    results. We have addressed the problem of finding significant unusually 
    subneworks, named motifs, overcoming to the consuming step of network 
    simulation by providing analytical models. Our algorithms are the basis 
    to understand the topological proprieties of biological systems. 
    In order to analyze their dynamics, we model signaling and metabolic 
    pathways by using languages, techniques, and tools well established in 
    the context of electronic design automation. Our intuition here is that 
    several characteristics and issues to model biological systems are 
    common to the electronics system modelling, such as concurrency, 
    reactivity, abstraction levels, as well as state space explosion 
    during validation. We study static and dynamic network characteristics 
    by associating to nodes profiles information from genomic and 
    trascriptomic data. We work both with bulk and single cell data. 
    Here we customize and develop pipelines for genomic variants detections 
    and annotation, for coding and non coding deferential analysis. 
    We also design predict models that from data expression construct 
    network of interactions in particular for small, circular, circulate 
    RNA interferences. We study new theoretical methods aimed at 
    investigating biological sequences by using information theory, 
    algorithmic approaches and advanced data structure engineering, 
    from DNA to RNA level. The goal is reached by defining mathematical 
    representations that characterize biological sequences and that allow 
    us to investigate single molecule properties as well as to examine the 
    composition of set of sequences. Our algorithms are developed sequential 
    and in several cases, also in parallel (multicore or GPU). Our studies 
    produce software and data available for the open research development 
    and distributed in the appropriate platforms (web based, Cytoscape and R).
  </div>
  


  <script type="text/javascript" src="js/mdb.min.js"></script>
  <!-- Custom scripts -->
  <script type="text/javascript"></script>
  </body>
</html>